source/crud/update.txt

```diff
@@ -0,0 +1,153 @@
+.. _mongosh-update:
+
+================
+Update Documents
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+The MongoDB shell provides the following methods to update documents in
+a collection:
+
+- To update a single document, use
+  :method:`db.collection.updateOne(\<filter\>, \<update\>, \<options\>)
```

Comment:
we don't include the parameters for the methods in the other tutorials. not sure we should include them here to be consistent.